lib/improver/tests/nowcasting/accumulation/test_Accumulation.py

```diff
@@ -186,48 +186,41 @@ def test_basic(self):
         returned. Also test that the returned list of cubes has the
         expected units."""
         expected_time_interval = 60
-        expected_integral = 10
         expected_cubes = self.cubes.copy()
         for cube in expected_cubes:
             cube.convert_units("m/s")
-        cubes, time_interval, integral = (
-            Accumulation()._check_inputs(self.cubes))
+        cubes, time_interval = Accumulation()._check_inputs(self.cubes)
         self.assertEqual(cubes, expected_cubes)
         self.assertEqual(time_interval, expected_time_interval)
-        self.assertEqual(integral, expected_integral)
 
     def test_specify_accumulation_period(self):
         """Test that the expected time interval and integral value is
         returned when the accumulation period is specified. Also test that the
         returned list of cubes has the expected units."""
         expected_time_interval = 60
-        expected_integral = 60
         expected_cubes = self.cubes.copy()
         for cube in expected_cubes:
             cube.convert_units("m/s")
         accumulation_period = 60*60
         plugin = Accumulation(accumulation_period=accumulation_period)
-        cubes, time_interval, integral = plugin._check_inputs(self.cubes)
+        cubes, time_interval = plugin._check_inputs(self.cubes)
         self.assertEqual(cubes, expected_cubes)
         self.assertEqual(time_interval, expected_time_interval)
-        self.assertEqual(integral, expected_integral)
         self.assertEqual(plugin.accumulation_period, accumulation_period)
 
     def test_specify_forecast_period(self):
         """Test that the expected time interval and integral value is
```

Comment:
The doc string needs to be updated to remove mention of the integral value. The others should be checked likewise.

Reply:
Updated docstrings to remove mention of integral value.